src/vl-input-addon.js

```diff
@@ -22,4 +35,20 @@ export const VlInputAddonElement = (SuperClass) => {
  */
 export class VlInputAddon extends VlInputAddonElement(HTMLParagraphElement) {}
 
-define('vl-input-addon', VlInputAddon, {extends: 'p'});
\ No newline at end of file
+/**
+ * VlButton-Input-addon
```

Comment:
Dashes mogen weg en addon moet met hoofdletter

```suggestion
 * VlButtonInputAddon
```